Revit_Core_Engine/Convert/Revit/FromRevit/View.cs

```diff
@@ -0,0 +1,135 @@
+                                                         .Select(kvp => new ViewFilterWithOverrides { Filter = kvp.Key, Overrides = kvp.Value }).ToList();
+
+            /*2. Create BHoM View Object with Name and FilterWithOverrides objects */
+            view = new oM.Adapters.Revit.Elements.View { Name = revitView.Name, FiltersWithOverrides = filtersWithOverrides };
+
+            //Set identifiers, parameters & custom data
+            view.SetIdentifiers(revitView);
+            view.CopyParameters(revitView, settings.MappingSettings);
+            view.SetProperties(revitView, settings.MappingSettings);
+
+            refObjects.AddOrReplace(revitView.Id, view);
+            return view;
+        }
+
+        /***************************************************/
+
+        private static OverrideGraphicSettings overrideGraphicSettingsFromRevit(this View element, Autodesk.Revit.DB.OverrideGraphicSettings revitOverrides)
+        {
+            // Initialize BHoM OverrideGraphicsSettings object
+            OverrideGraphicSettings overrideGraphicsSettings = new OverrideGraphicSettings();
+
+            // Convert COLORS 
+            overrideGraphicsSettings.LineColor = System.Drawing.Color.FromArgb(revitOverrides.CutLineColor.Red, revitOverrides.CutLineColor.Green, revitOverrides.CutLineColor.Blue);
+            overrideGraphicsSettings.CutColor = System.Drawing.Color.FromArgb(revitOverrides.CutBackgroundPatternColor.Red, revitOverrides.CutBackgroundPatternColor.Green, revitOverrides.CutBackgroundPatternColor.Blue);
+            overrideGraphicsSettings.SurfaceColor = System.Drawing.Color.FromArgb(revitOverrides.SurfaceBackgroundPatternColor.Red, revitOverrides.SurfaceBackgroundPatternColor.Green, revitOverrides.SurfaceBackgroundPatternColor.Blue);
+
+            // Convert LINE PATTERNS
+            String linePatternName = element.Document.GetElement(revitOverrides.CutLinePatternId).Name;
+
+            if (linePatternName.ToUpper().Contains("DASH")) { overrideGraphicsSettings.LinePattern = oM.Revit.Enums.LinePattern.DASH; }
+            else if (linePatternName.ToUpper().Contains("DOT")) { overrideGraphicsSettings.LinePattern = oM.Revit.Enums.LinePattern.DOT; }
+            else if (linePatternName.ToUpper().Contains("HIDDEN")) { overrideGraphicsSettings.LinePattern = oM.Revit.Enums.LinePattern.HIDDEN; }
+            else if (linePatternName.ToUpper().Contains("SOLID")) { overrideGraphicsSettings.LinePattern = oM.Revit.Enums.LinePattern.SOLID; }
+            else { BH.Engine.Base.Compute.RecordWarning($"The Revit Line Pattern {linePatternName} is not implemented yet in the BHoM.\n By default, the Line Pattern {linePatternName} will be set to SOLID."); }
+
+
+            // Convert CUT PATTERNS
+            String cutPatternName = element.Document.GetElement(revitOverrides.CutBackgroundPatternId).Name;
```

Comment:
You need to address the case where `CutLinePatternId == -1` (and all other Ids pointing at patterns, linestyles etc.)